subsys/bootloader/bootloader.c

```diff
@@ -86,9 +86,6 @@ void uninit_used_peripherals(void)
 }
 
 #ifdef CONFIG_SW_VECTOR_RELAY
-#ifndef CONFIG_SB_C_RUNTIME_SETUP_VARIANT_ZEPHYR
-_GENERIC_SECTION(.vt_pointer_section) u32_t _vector_table_pointer;
```

Comment:
Are you sure this line is not needed?
It's for nRF51 which has no VTOR.

Reply:
I could be wrong but I thought it was not needed if `SB_C_RUNTIME_SETUP_VARIANT_ZEPHYR` is standard and that it defines this `extern  _vector_table_pointer` in some other file am I wrong?

Reply:
Oh, sorry I misread, it's #ifndef not #ifdef.